### PR TITLE
[Done] Attendant Login - Will add keyboard seperately

### PR DESCRIPTION
**Pull request by RowanDavitt**
I think I have to wait for the keyboard to be done.

Everything else should work at the moment

I'm not really sure if the login information should be a username and password or just a password (currently it's just a password)

The changes I've made are:
- created AttendantLoginScreen
- created AttendantPane (similar to customerStation pane) to make it easier to manage multiple screens for the attendant GUI (may not be nessacery since there's only two screens at the moment)
- changed AttendantStationGUI to open the AttendantPane first which will display AttendantLoginScreen
- added login() and an ArrayList called logins to AttendantControl
- added  addFakeAttendantLogin in FakeDataInitializer to add arbitrary login info
- added tests for addFakeAttendantLogin in FakeDataInitializertest
- made tests for login() in testAttendantControl
- added loggedIn() to the AttendantControlListener (this caused a small change for a lot of other files that implemented the listener)

A bunch of binary files also were committed and I don't know why

### CODE REVIEW

**Review by liza-khr — COMMENTED** [review]
I think we don't need to wait for the keyboard to be finished; we can just merge this into main (once the conflicts are resolved) and the keyboard can be added after it is finished

**Comment by RowanDavitt** [comment]
> I think we don't need to wait for the keyboard to be finished; we can just merge this into main (once the conflicts are resolved) and the keyboard can be added after it is finished

Ok, the conflicts should up cleared up now.

**Review by liza-khr — DISMISSED** [review]

**Review by liza-khr — DISMISSED** [review]

**Comment by hackrmomo** [comment]
Should be good to go I think

**Review by hackrmomo — APPROVED** [review]

**Review by liza-khr — APPROVED** [review]

**Pull request merged**
